airbyte_cdk/test/standard_tests/_job_runner.py

```diff
@@ -56,9 +56,9 @@ def spec(self, logger: logging.Logger) -> Any:
 
 def run_test_job(
     connector: IConnector | type[IConnector] | Callable[[], IConnector],
-    verb: Literal["read", "check", "discover"],
-    test_scenario: ConnectorTestScenario,
+    verb: Literal["spec", "read", "check", "discover"],
     *,
+    test_scenario: ConnectorTestScenario | None = None,
```

Comment:
This now handles a "None" scenario - especially for the `spec` use case. If test_scenario is omitted, we presumably won't have a `config` but this is not needed for `spec` anyway, and there may be additional cases where it is not needed which we haven't considered as of yet.